packages/node_modules/@ciscospark/react-redux-spark/src/component.js

```diff
@@ -31,10 +31,16 @@ const injectedPropTypes = {
 };
 
 const propTypes = {
-  accessToken: PropTypes.string.isRequired,
+  accessToken: PropTypes.string,
+  jwtToken: PropTypes.string,
   ...injectedPropTypes
 };
 
+const defaultProps = {
+  accessToken: '',
+  jwtToken: ''
```

Comment:
do we want to use the term `jwtToken`. We should probably match the term used my the dev portal and hydra, which I think may still be something like `guestId`?

Reply:
Good call, I'll change it to match the portal which calls it "Guest Token"